.github/workflows/build.yaml

```diff
@@ -202,64 +202,23 @@ jobs:
               uses: actions/checkout@v2
               with:
                   submodules: true
-            # - name: Initialize CodeQL
-            #   if: ${{ github.event_name == 'push' && github.event.ref == 'refs/heads/master' }}
-            #   uses: github/codeql-action/init@v1
-            #   with:
-            #       languages: "cpp"
-
```

Comment:
This change is unrelated to the state purpose of this PR, and makes it harder to re-enable CodeQL later as needed.  Why is it being made?

Reply:
This is for python_lib only. The comments seem just noise.

Generally I would actually prefer to remove these commented out line everywhere because later we can add them back (one can look at commit history), however in this case I figured in particular python likely does not need this.